.circleci/config.yml

```diff
@@ -0,0 +1,231 @@
+          key: conda-py37-v1-{{ checksum "setup.cfg" }}
+      - run:
+          name: Run integration tests
+          no_output_timeout: 10m
+          command: |
+            apt-get install -yqq make
+            source activate phys2denoise_py37  # depends on makeenv_37
+            make integration
+            mkdir /tmp/src/coverage
+            mv /tmp/src/phys2denoise/.coverage /tmp/src/coverage/.coverage.integration37
+      - store_artifacts:
+          path: /tmp/data
+      - persist_to_workspace:
+          root: /tmp
+          paths:
+              - src/coverage/.coverage.integration37
+
+  style_check:
+    docker:
+      - image: continuumio/miniconda3
+    working_directory: /tmp/src/phys2denoise
+    steps:
+      - checkout
+      - restore_cache:
+          key: conda-py37-v1-{{ checksum "setup.cfg" }}
+      - run:
+          name: Style check
+          command: |
+            apt-get install -yqq make
+            source activate phys2denoise_py37  # depends on makeenv37
+            make lint
+      - store_artifacts:
+          path: /tmp/data/lint
+
+  build_docs:
+    working_directory: /tmp/src/phys2denoise
+    docker:
+      - image: continuumio/miniconda3
+    steps:
+      - attach_workspace:  # get phys2denoise
+          at: /tmp
+      - restore_cache:  # load environment
+          key: conda-py37-v1-{{ checksum "setup.cfg" }}
+      - run:
+          name: Build documentation
+          command: |
+            apt-get install -yqq make
+            source activate phys2denoise_py37  # depends on makeenv_37
+            make -C docs html
+      - store_artifacts:
+          path: /tmp/src/phys2denoise/docs/_build/html
+
+  merge_coverage:
+    working_directory: /tmp/src/phys2denoise
+    docker:
+      - image: continuumio/miniconda3
+    steps:
+      - attach_workspace:
+          at: /tmp
+      - checkout
+      - restore_cache:
+          key: conda-py37-v1-{{ checksum "setup.cfg" }}
+      - run:
+          name: Merge coverage files
+          command: |
+            apt-get install -yqq curl
+            source activate phys2denoise_py37  # depends on makeenv37
+            cd /tmp/src/coverage/
+            coverage combine
+            coverage xml
+      - store_artifacts:
+          path: /tmp/src/coverage
+      - codecov/upload:
+          file: /tmp/src/coverage/coverage.xml
+
+workflows:
+  version: 2.1
+  build_test:
+    jobs:
+      - makeenv_37
+      # - unittest_36
+      # - unittest_37:
+      #     requires:
+      #       - makeenv_37
+      # - integrationtest_36
+      # - integrationtest_37:
+      #     requires:
+      #       - makeenv_37
+      - style_check:
+          requires:
+            - makeenv_37
+      # - build_docs:
+      #     requires:
+      #       - makeenv_37
+      - merge_coverage:
+          requires:
+            # - unittest_36
+            # - unittest_37
+            # - integrationtest_36
+            # - integrationtest_37
```

Comment:
Why don't we remove the commented lines for now? Just to make sure they're not causing any issue.